package.json

```diff
@@ -13,6 +13,11 @@
   "dependencies": {
     "@contentful/rich-text-react-renderer": "^15.2.0",
     "@contentful/rich-text-types": "^15.1.0",
+    "@testing-library/dom": "^8.7.1",
+    "@testing-library/jest-dom": "^5.14.1",
+    "@testing-library/react": "^12.1.1",
+    "@testing-library/user-event": "^13.2.1",
+    "ansi-regex": "^6.0.1",
```

Comment:
I'm curious about your reasoning for adding these here? I asked @pydanny about testing dependencies just now being in `dependencies` vs. `devDependencies` and he recommended just the latter. I'm open to this if there's a good reason for it, though.

Reply:
These shouldn't be here - these are dev dependencies and I'm not sure why the `ansi-regex` was added here. The dependency that is using it will install it and we don't have to.